src/main/webapp/app/exercises/programming/manage/update/update-components/theia/programming-exercise-theia.component.ts

```diff
@@ -0,0 +1,77 @@
+import { Component, Input, OnChanges, SimpleChanges } from '@angular/core';
+import { ProgrammingExercise, ProgrammingLanguage } from 'app/entities/programming-exercise.model';
+import { ProgrammingExerciseCreationConfig } from 'app/exercises/programming/manage/update/programming-exercise-creation-config';
+import { TheiaService } from 'app/exercises/programming/shared/service/theia.service';
+
+@Component({
+    selector: 'jhi-programming-exercise-theia',
+    templateUrl: './programming-exercise-theia.component.html',
+    styleUrls: ['../../../programming-exercise-form.scss'],
```

Comment:
We want to make our new components stand alone:
https://docs.artemis.cit.tum.de/dev/guidelines/client.html#components

Reply:
done